parser/js/test/unit/jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    displayName: 'Unit',
+    transform: {
+        "^.+\\.ts": "ts-jest",
+    },
+    testRegex: "(.*|(\\.|/)(test|spec))\\.(js|ts)$",
+    moduleFileExtensions: ["ts", "js", "json", "node"],
+};
```

Comment:
@ebrelsford Surprisingly, I noticed that this introduces another failure when running:

```
npm test
```

```
 FAIL   Unit  test/unit/jest.config.js
  ● Test suite failed to run

    Your test suite must contain at least one test.

      at onResult (../../node_modules/@jest/core/build/TestScheduler.js:133:18)
      at ../../node_modules/@jest/core/build/TestScheduler.js:254:19
      at ../../node_modules/emittery/index.js:363:13
          at Array.map (<anonymous>)
      at Emittery.emit (../../node_modules/emittery/index.js:361:23)
```

Reply:
Ah yeah I overcomplicated the jest config here, simplifying it should fix--thanks for pointing that out.

Reply:
[6732f66](https://github.com/maplibre/maplibre-tile-spec/pull/49/commits/6732f660b627b731716ce3d7f316f2951d8079b9) looks good 👍🏼